metagraph/src/graph/alignment/aligner_extender_methods.cpp

```diff
@@ -699,9 +728,11 @@ ::update_columns(NodeType incoming_node,
         return;
 
     // set boundaries for vertical band
-    auto *incoming = &dp_table.find(incoming_node).value();
+    auto incoming_find = dp_table.find(incoming_node);
+    auto *incoming = &incoming_find.value();
 
     if (dp_table.size() == 1 && out_columns.size() && out_columns.front().first != incoming_node) {
+        dp_table.expand_to_cover(incoming_find, 0, size);
```

Comment:
maybe `... = *dp_table.find(incoming_node);`?

Reply:
I simplified this a bit.